docs/Reference/Pantheon-API-Methods.md

```diff
@@ -3738,6 +3738,36 @@ None
     The `EEA` API methods are not enabled by default for JSON-RPC. Use the [`--rpc-http-api`](Pantheon-CLI-Syntax.md#rpc-http-api) 
     or [`--rpc-ws-api`](Pantheon-CLI-Syntax.md#rpc-ws-api) options to enable the `EEA` API methods.
 
+### eea_getPrivacyPrecompileAddress
+
+Returns the address to which the [privacy pre-compiled contract](../Privacy/Private-Transaction-Processing.md) is mapped. 
```

Comment:
Returns the address of the privacy pre-compiled contract.

Reply:
Fixed